Jenkinsfile

```diff
@@ -76,8 +76,18 @@ def s3_upload(type, flavor, os, arch) {
   sh "mv ${buildFolder}/${packageFile} ${buildFolder}/${renamedFile}"
   packageFile = renamedFile
 
+  def buildType = sh (
+    script: "cat BUILDTYPE",
+    returnStdout: true
+  ).trim().toLowerCase()
+
+  def buildDest =  "s3://rstudio-ide-build/${flavor}/${os}/${arch}/"
+  if (buildType != "daily") {
+    buildDest =  "s3://rstudio-ide-build-internal/${buildType}/${flavor}/${os}/${arch}/"
```

Comment:
Nit: we put this into a bucket called "internal", but I assume these will still be user-visible (just from elsewhere)?

Reply:
These will only become user visible when we upload them as we do with previews and releases